libs/product/src/facades/composite-product/composite-product.facade.ts

```diff
@@ -22,67 +22,156 @@ export class DaffCompositeProductFacade<T extends DaffProduct = DaffProduct> imp
 	selectors = getDaffProductSelectors<T>();
 	
 	constructor(private store: Store<DaffProductReducersState<T>>) {}
-	
+
+	/**
+	 * Get the minimum price for a composite product, excluding optional items and regardless of applied options.
```

Comment:
Aren't these documented on the interface? I say its better to configure dgeni to inherit the docs from the interface or just direct people to the interface.

Reply:
Dgeni is pretty hellish to try to bend to your will. I think we could make the docs have a link to the interface in the overall documentation for this file, but configuring dgeni to inherit the docs from the interface would be really tricky

Reply:
Yeah I think its preferable to have [a link](https://github.com/angular/dgeni-packages#inline-tag-definitions) back to the interface field so we only have to maintain those docs in one place.

Reply:
done. Though I did leave the doc for the `dispatch` function, since this is hidden behind an `extends` in the facade interface. I think this could pose a problem with this documentation pattern if there are multiple levels of interface extension.

Reply:
I think you can use `{@link some/uri Some Title}` to link to specific facade interface fields, not sure though. I thought we could link back to the interface fields for each of the facade fields.

Reply:
Wouldn't we need to create anchors to the places the links go to? And those anchors would need ids. Then we'd need to upkeep the links to go to the correct ids, so we'd end up with the same problem of extra maintenance. Though I guess we wouldn't need to maintain the actual descriptions.

Reply:
According to the docs that I linked ^^^, the dgeni jsdoc package can automatically replace link tags with anchors. I think the ID would just be the name of the field so I don't think that would really need any maintenance unless we refactor the name of the field. And in that case a search and replace should pick up the field name in the comments.

Reply:
I looked into this and it looks doable, since there are examples in the angular material repo. I'll need to configure some things in dgeni, so I'd like to do that in another PR. Added an issue for it here: https://github.com/graycoreio/daffodil/issues/1102

Reply:
sounds good